### ISSUE TITLE
Grid get 0 width when inside an OrderedLayout

### ISSUE THREAD

**Issue by pekam** [vaadin/vaadin-grid-flow#11]
@caalador commented on [Fri Sep 29 2017](https://github.com/vaadin/flow/issues/2543)

When adding Grid inside a layout (Vertical/Horizontal/Flex) the component gets a 0px width.

```
        Grid<Person> grid = new Grid<>();
        grid.setItems(createItems());

        grid.addColumn("Name", Person::getName);
        grid.addColumn("Age", person -> Integer.toString(person.getAge()));
grid.setSelectionMode(Grid.SelectionMode.NONE);
        grid.setId("none-selection");
        HorizontalLayout layout = new HorizontalLayout(grid);
        addCard("Grid with No Selection Enabled", layout);
```

**Comment by pleku** [comment]
@gilberto-torrezan can this be now closed ?

**Comment by gilberto-torrezan** [comment]
This issue was fixed with the Grid 5.0.0.beta3 release.
